app/src/main/java/com/ataulm/artcollector/gallery/ui/GalleryViewModel.kt

```diff
@@ -1,54 +1,68 @@
 package com.ataulm.artcollector.gallery.ui
 
+import android.view.View
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.paging.*
 import com.ataulm.artcollector.Event
+import com.ataulm.artcollector.Painting
 import com.ataulm.artcollector.gallery.domain.GetGalleryUseCase
-import kotlinx.coroutines.*
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
 import javax.inject.Inject
 
 internal class GalleryViewModel @Inject constructor(
         private val getGallery: GetGalleryUseCase
 ) : ViewModel() {
 
-    private val _gallery = MutableLiveData<UiGallery>()
-    val gallery: LiveData<UiGallery> = _gallery
-
     private val _events = MutableLiveData<Event<NavigateCommand>>()
     val events: LiveData<Event<NavigateCommand>>
         get() = _events
 
     private val parentJob = Job()
     private val coroutineScope = CoroutineScope(parentJob)
```

Comment:
this can be updated with some "new" dependencies  - see the comment below with the TODO